website/api/zkvm/install.md

```diff
@@ -25,6 +25,8 @@ cargo binstall cargo-risczero
 cargo risczero install
 ```
 
+> Note: To install a specific version instead of using the latest stable version, use `cargo binstall cargo-risczero@<version>` where the version is the crates.io release tag. 
```

Comment:
```suggestion
> Note: To install a specific version instead of using the latest stable version, use `cargo binstall cargo-risczero@<version>` where the version is a [release tag](https://github.com/risc0/risc0/releases). 
```